js-runner/webpack.common.js

```diff
@@ -11,33 +11,37 @@ const common = {
           loader: "babel-loader",
           options: {
             presets: [],
-            plugins: ["transform-object-rest-spread"],
+            plugins: ["@babel/plugin-proposal-object-rest-spread"],
           },
         },
       },
     ],
   },
-  plugins: [
-    new webpack.DefinePlugin({
-      __DEBUG: JSON.stringify(false),
-    }),
-  ],
```

Comment:
This was throwing an error because both prod & dev files add the plugin themselves.